docs/integrations/azure/org.md

```diff
@@ -258,21 +258,96 @@ The `myUserTransformer`, `myGroupTransformer`, `myOrganizationTransformer`, and
 import { GroupEntity, UserEntity } from '@backstage/catalog-model';
+import { createBackendModule } from '@backstage/backend-plugin-api';
+
+// The Group transformer transforms Groups that are ingested from MS Graph
+export async function myGroupTransformer(
+  group: MicrosoftGraph.Group,
+  groupPhoto?: string,
+): Promise<GroupEntity | undefined> {
+  const backstageGroup = await defaultGroupTransformer(group, groupPhoto);
+  return backstageGroup;
+}
+
+// The User transformer transforms Users that are ingested from MS Graph
+export async function myUserTransformer(
+  graphUser: MicrosoftGraph.User,
+  userPhoto?: string,
+): Promise<UserEntity | undefined> {
+  const backstageUser = await defaultUserTransformer(graphUser, userPhoto);
+  return backstageUser;
+}
+
+// The Organization transformer transforms the root MS Graph Organization into a Group
+export async function myOrganizationTransformer(
+  graphOrganization: MicrosoftGraph.Organization,
+): Promise<GroupEntity | undefined> {
+  const backstageOrg = await defaultOrganizationTransformer(graphOrganization);
+  return backstageOrg;
+}
+
+// The Provider Config transformer enables modification of the plugin config
+export async function myProviderConfigTransformer(
+  provider: MicrosoftGraphProviderConfig,
+): Promise<MicrosoftGraphProviderConfig> {
+  return provider;
+}
+
+// Wrapping these functions in a Module allows us to inject them into the Catalog plugin easily
+export const myMsgraphTransformersModule = createBackendModule({
+  pluginId: 'catalog',
+  moduleId: 'msgraph-org',
+  register(reg) {
+    reg.registerInit({
+      deps: {
+        microsoftGraphTransformers:
+          microsoftGraphOrgEntityProviderTransformExtensionPoint,
+      },
+      async init({ microsoftGraphTransformers }) {
+        // Set the transformers to our custom functions
+        microsoftGraphTransformers.setUserTransformer(myUserTransformer);
+        microsoftGraphTransformers.setGroupTransformer(myGroupTransformer);
+        microsoftGraphTransformers.setOrganizationTransformer(
+          myOrganizationTransformer,
+        );
+        microsoftGraphTransformers.setProviderConfigTransformer(
+          myProviderConfigTransformer,
+        );
+      },
+    });
+  },
+});
+
+// Export a default to make importing into the backend simpler
+export default myMsgraphTransformersModule;
+```
 
-// This group transformer completely replaces the built in logic with custom logic.
+Now lets customize each of the providers to suit our needs.
+
+The Group Transformer will have the default logic completely removed and replaced with our custom logic:
+
+```ts
 export async function myGroupTransformer(
   group: MicrosoftGraph.Group,
   groupPhoto?: string,
 ): Promise<GroupEntity | undefined> {
+  // highlight-remove-start
+  const backstageGroup = await defaultGroupTransformer(group, groupPhoto);
+  return backstageGroup;
+  // highlight-remove-end
+  // highlight-add-start
   return {
```

Comment:
We do an if check in the User example below, should we do one here as well?

Reply:
We are removing the call to `defaultGroupTransformer` to apply our own logic, so no need to check if it returns.